simulation/sub8_gazebo/nodes/fake_perception.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+#TODO
+#Get modularity working
+#Generate Services using a for loop
+#Step 1: Parse Launch File into this so I can read new models and coords directly from there
+#Step 2: Figure out which service is accessing master pose function
+#Step 3: Making master junk functions
+#Step 4: Make the whole thing a class <<< This should come first.
+import cv2
+from geometry_msgs.msg import Pose, Point, Quaternion, PoseStamped, Vector3Stamped, Vector3
+from mil_misc_tools import text_effects
+from mil_msgs.srv import SetGeometry
+import numpy as np
+import rospy
+from std_srvs.srv import SetBool, SetBoolResponse
+from sub8_msgs.srv import VisionRequestResponse, VisionRequest, VisionRequest2D, VisionRequest2DResponse
+from std_msgs.msg import Header, Float64
+import os
+import rospkg
+
+import sys
+import time
+from tf import transformations
+import yaml
+import xml.etree.ElementTree
+rospack = rospkg.RosPack()
+#Find a way to parse through the yaml correctly. Might be unneccessary but nice.
+config_file = os.path.join(rospack.get_path('sub8_missions'), 'sub8', 'vision_proxies.yaml')
+f = yaml.load(open(config_file, 'r'))
+launch_file = os.path.join(rospack.get_path('sub8_gazebo'), 'launch', 'duck.launch')
+launch = xml.etree.ElementTree.parse(launch_file).getroot()
+
+fprint = text_effects.FprintFactory(title="SIMULATOR").fprint
+
+def handle_fake_perception(extra,resp):
```

Comment:
You should give functions less ambigous names and/or leave a header comment in each function like this:
```
def handle_fake_perception(extra,resp):
    '''
    Parses the gazebo launch file and returns a vision response for this target
    @param extra Does x
    @param resp Does y
    '''


```